### DISCUSSION TITLE
Endpoint: supported versions

### DISCUSSION THREAD

**Discussion by keunes** [OpenPodcastAPI/api-specs#11] [Problem definitions & solutions]
From https://github.com/OpenPodcastAPI/api-specs/discussions/10#discussioncomment-5008208

> We'll … need an endpoint about supported versions (e.g. a json array that contains all supported versions). [matrix](https://spec.matrix.org/v1.6/client-server-api/#server-discovery) does this after the discovery, it has an endpoint `https://$host/$prefix/versions`. This might be our way-to-go as well, since this way it wouldn't even be necessary to put these info inside the .well-known file. That way we could take care of that endpoint at a later time.

**Comment by georgkrause** [comment]
I think a versions endpoint is cheap to implement, we should go this path. We only need to decide on a version format, I guess.

@Sporiff @JonOfUs How about `v1` `v2` ..., only allowing integers
